go.mod

```diff
@@ -273,3 +275,5 @@ replace github.com/google/gnostic => github.com/googleapis/gnostic v0.6.9
 replace gopkg.in/alecthomas/kingpin.v2 => github.com/alecthomas/kingpin v1.3.8-0.20210301060133-17f40c25f497
 
 replace github.com/sercand/kuberesolver => github.com/sercand/kuberesolver/v5 v5.1.1
+
+replace google.golang.org/grpc => google.golang.org/grpc v1.57.2
```

Comment:
I have to pin grpc version to 1.57.2 because https://github.com/thanos-io/thanos/pull/7078 breaks our build.

Reply:
OK... This is not needed anymore. I just pinned kuberesolver to v5